### DISCUSSION TITLE
getting volume does not exit - status code 400

### DISCUSSION THREAD

**Discussion by thinksabin** [vmware-tanzu/velero#5514] [Community support Q&A]
Hi there,

any idea how to fix an error where it complains around the volume doesnt exit?
`time="2022-10-28T11:26:31Z" level=error msg="Error backing up item" backup=velero/k8-sq-backup-2-28oct2022 error="error getting volume info: rpc error: code = Unknown desc = InvalidVolume.NotFound: The volume 'vol-0a8421e545fd87dfc' does not exist.\n\tstatus code: 400, request id: 8b9d9845-5e5f-4c41-98e8-b490e632a377" logSource="pkg/backup/backup.go:435" name=data-postgres-postgresql-0`

how can I change the volume id its complaining about. That volume is not in use for sure but I dont know why new volume is not being backed up. 

Thanks

**Comment by draghuram** [comment]
Are you sure other resources (including volumes) are not being backed up? It would help if you can provide more information. For example, are you doing just resource backup or restic backup as well? What resources are included in the backup?

**Comment by thinksabin** [comment]
Hi,
I'm doing resource backup only no restic backup.
I managed to debug it. I found out the volume mentioned in the pvc was not present as mentioned by the error. Found out this error is taking place in the particular namespace backup only and when I do resource backup of everything. In general one particular namespace only where pvc and pv were present but the volume doesnt exist. I dont know how I came into this situation but as a simple solution I just deleted the pvc and pv since it wasnt being used by any pods. Later velero worked fine. 
Thanks.
